nservicebus/service-fabric/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Service Fabric Persistence
+reviewed: 2017-03-20
+component: ServiceFabricPersistence
+---
+
+A persister built on top of [Service Fabric Reliable Collections](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-reliable-collections).
+
+
+## Usage
+
+Add a NuGet package reference to `NServiceBus.Persistence.ServiceFabric`. Configure the persistence technology using the following configuration API.
+
+snippet: ServiceFabricPersistenceConfiguration
+
+## Limitations
+
+### Storage types
+
+Currently saga and outbox storage types are supported. For timeouts, subscriptions or gateway deduplication use either the native capability of the transport of choice or another persistence such as [Azure Storage Persistence](/nservicebus/azure-storage-persistence) or [In-Memory persistence](/nservicebus/persistence/in-memory.md).
+
+### Viewing the data
+
+Currently, Service Fabric does not provide capabilities to view the data that is stored inside reliable collections. The data has to be programmatically accessed and exposed over the [backup and restore APIs](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-reliable-services-backup-restore).
```

Comment:
I don't know why this has been changed. The backup and restore APIs are for backup and restore. In order to visualize the data outside you have to basically host an HTTP endpoint that exposed it. Do we really want to promote backup and restore APIs as a solution for viewing the data?